src/index.ts

```diff
@@ -611,19 +622,20 @@ function sendPartialResponse(
   pretty: boolean,
   response: Response,
   result: FormattedExecutionResult | FormattedExecutionPatchResult,
+  hasNext: boolean,
+  initial: boolean = false,
 ): void {
   const json = JSON.stringify(result, null, pretty ? 2 : 0);
   const chunk = Buffer.from(json, 'utf8');
-  const data = [
-    '',
-    '---',
-    'Content-Type: application/json; charset=utf-8',
-    'Content-Length: ' + String(chunk.length),
-    '',
-    chunk,
-    '',
-  ].join('\r\n');
-  response.write(data);
+  const data = ['Content-Type: application/json; charset=utf-8', '', chunk];
+  if (initial) {
+    data.unshift('---');
+  }
+  data.unshift('');
+  if (hasNext) {
```

Comment:
Can you read `hasNext` off of the `result` instead of passing it in as a separate arguments?

Reply:
Yeah... haha 🤦‍♂️ that is better actually. Simpler code too.